core/lib/spree/permission_sets/restricted_stock_management.rb

```diff
@@ -2,6 +2,11 @@
 
 module Spree
   module PermissionSets
+    # Full permissions for stock management limited to allowed locations.
+    #
+    # This permission set grants full control over all stock items a user has
```

Comment:
What determines if a user has access to a specific stock item?

Reply:
I rephrased it a bit, let me know what you think :slightly_smiling_face: